### PR TITLE
⚗️ Adjust buttons' order in error page

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: src/experiments/civic/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export default {
   name: 'civic-register-page-error',
   experimentID: '-ZAD0MyxTySnOmA902g6Jg',
-  isEligible: ({ route }) => !process.server && route.name === 'civic-register',
+  isEligible: () => !process.server,
```
Comment (williamchong007) [inline src/experiments/civic/index.js]: if only want to test ppl directly landed on `/civic`, can keep the route check with name `civic`

Comment (nwingt) [reply src/experiments/civic/index.js]: reverted

Comment (williamchong007) [reply src/experiments/civic/index.js]: `civic` not `civic-register`

Comment (williamchong007) [reply src/experiments/civic/index.js]: this `isEligible` hook only trigger first time ppl arrive the site, so its kind of strange

Comment (nwingt) [reply src/experiments/civic/index.js]: updated

**Pull request merged**